tests/test_utils.py

```diff
@@ -40,6 +40,18 @@ def test_get_rebuilt_nvr(mock_time, rebuilt_nvr_release_suffix):
     assert rebuilt_nvr == expected
 
 
+@patch("freshmaker.config.Config.unpublished_exceptions",
+       new_callable=PropertyMock,
+       return_value=[{"repository": "some-repo", "registry": "some-registry"}])
+def test_is_in_unpublished_exceptions(exception):
+    images = [{"repositories": [{"registry": "some-registry", "repository": "repo"}]},
+              {"repositories": [{"registry": "registry", "repository": "some-repo"}]},
+              {"repositories": [{"registry": "some-registry", "repository": "some-repo"}]}]
+
+    results = list(map(is_in_unpublished_exceptions, images))
+    assert results == [False, False, True]
```

Comment:
```python
    assert is_in_unpublished_exceptions({"repositories": [{"registry": "some-registry", "repository": "repo"}]}) is False
    assert is_in_unpublished_exceptions({"repositories": [{"registry": "registry", "repository": "some-repo"}]}) is False
    assert is_in_unpublished_exceptions({"repositories": [{"registry": "some-registry", "repository": "some-repo"}]}) is True
```

But it's a good candidate to use `@pytest.mark.parametrize`.

Reply:
When you use `@pytest.mark.parametrize` it generates a new test for every parameter, so with 3 parameters you will get 3 tests executed and addressed in the output. I prefer to have one output for one tested functionality so I would not use this fixture here. And secon